text_generation/causal_lm/cpp/speculative_decoding_lm.cpp

```diff
@@ -119,10 +119,10 @@ int main(int argc, char* argv[]) try {
     core.add_extension(OPENVINO_TOKENIZERS_PATH);  // OPENVINO_TOKENIZERS_PATH is defined in CMakeLists.txt
     // tokenizer and detokenizer work on CPU only
     ov::InferRequest tokenizer = core.compile_model(
-        std::string{argv[1]} + "/openvino_tokenizer.xml", "CPU").create_infer_request();
+        std::string{argv[1]} + "/openvino_tokenizer/openvino_tokenizer.xml", "CPU").create_infer_request();
     auto [draft_input_ids, draft_attention_mask] = tokenize(tokenizer, argv[3]);
     ov::InferRequest detokenizer = core.compile_model(
-        std::string{argv[1]} + "/openvino_detokenizer.xml", "CPU").create_infer_request();
+        std::string{argv[1]} + "/openvino_tokenizer/openvino_detokenizer.xml", "CPU").create_infer_request();
```

Comment:
@apaniukov 
why extra subfolder is created? and why is it names `openvino_tokenizer` not just `tokenizer` like here https://huggingface.co/SimianLuo/LCM_Dreamshaper_v7/tree/main/tokenizer ?

Reply:
`tokenizer` and `tokenizer_2` are from a stable diffusion repo structure. The `openvino_tokenizer` folder is just an arbitrary name: https://github.com/huggingface/optimum-intel/pull/580#discussion_r1557865119, can be changed at any time.

Reply:
I would keep folders structure as in HF